svir/dialogs/viewer_dock.py

```diff
@@ -831,22 +929,49 @@ def write_export_file(self, filename):
                         line += "," + ",".join([
                             str(value) for value in values])
                     csv_file.write(line + os.linesep)
+            elif self.output_type == 'agg_curves-rlzs':
+                num_rlzs = self.agg_curves['array'].shape[1]
```

Comment:
A comment on the shape of the array would be useful (it is (P, R) for rlzs and (P, S) for stats) being

P = number of return periods
R  = number of realizations
S = number of statistics

Reply:
Done